database/seed.ts

```diff
@@ -0,0 +1,185 @@
+    const departmentData: (typeof department.$inferInsert)[] = []
+    const employeeData: (typeof employee.$inferInsert)[] = []
+    const schedulingData: (typeof scheduling.$inferInsert)[] = []
+
+    // Create number of user entries for each condominium
+    for (let i: number = 0; i < generateForEach; i++){
+        condominiumUserData.push({
+            id: faker.string.uuid(),
+            email: faker.internet.email(),
+            userType: userType.enumValues[1]
+        })
+    }
+
+    // Create a predefined number of entries for each condominium
+    condominiumUserData.map(condominiumUser => {
+        condominiumData.push({
+            id: faker.string.uuid(),
+            cnpj: faker.string.numeric({length: 13}),
+            name: faker.person.fullName(),
+            address: faker.location.streetAddress(),
+            cep: faker.location.zipCode({format: "########"}),
+            uf: faker.location.state({abbreviated: true}),
+            city: faker.location.city(),
+            userId: condominiumUser.id
+        })
+    })
+
+    // Create a predefined number of entries for each condominium
+    condominiumData.map(condominium => {
+        for(let i: number = 0; i < generateForEach; i++){
+            companyData.push({
+                id: faker.string.uuid(),
+                name: faker.company.name(),
+                cnpj: faker.string.numeric({length: 14}),
+                active: true,
+                cpf: faker.string.numeric({length: 11}),
+                condominiumAddress: faker.location.streetAddress(),
+                condominiumId: condominium.id
+            })
+        }
+    })
+
+    // Create a predefined number of departments for each company
+    companyData.map(company => {
+        for(let i: number = 0; i < generateForEach; i++) {
+            departmentData.push({
+                id: faker.string.uuid(),
+                name: faker.commerce.department(),
+                companyId: company.id
+            })
+        }
+    })
+
+    // create a predefine number of employees for each company for each department of this company
+    companyData.map(company => {
+        const departments = departmentData.filter(department => department.companyId === company.id)
+
+        departments.map(department => {
+
+            for(let i: number = 0; i < generateForEach; i++) {
+                const employeeUser = {
+                    id: faker.string.uuid(),
+                    userType: userType.enumValues[0],
+                    email: faker.internet.email()
+                }
+                employeeUserData.push(employeeUser)
+
+                employeeData.push({
+                    id: faker.string.uuid(),
+                    companyId: company.id,
+                    departmentId: department.id,
+                    name: faker.person.fullName(),
+                    email: faker.internet.email(),
+                    phoneNumber: faker.phone.number("55###########"),
```

Comment:
`faker-js/faker` is currently in the process of deprecating the usage of `phone.number(<string_pattern>)`. They instead recommend using either `string.numeric`, or a Regular Expression.

We don't really care much about the validity of those phone numbers for test data, so I believe prefixing the country code and the ninth digit from Brazil's phone numbers with subsequent 8 character long string of random numbers should be enough.

```suggestion
                    phoneNumber: `559${faker.string.numeric(8)}}`,
```